components/org.wso2.carbon.identity.api.user.organization/org.wso2.carbon.identity.rest.api.user.organization.v1/src/main/resources/user.organization.yaml

```
@@ -163,6 +164,14 @@ components:
       schema:
         type: boolean
         default: false
+    authorizedAppNameQueryParam:
+        in: query
+        name: authorizedAppName
+        required: false
+        description:
+          Retrieves authorized organizations for a user with a role in the given application.
```

Retrieves the organizations that are authorized for the user through role bound to the application.
```suggestion
          Retrieves authorized organizations for a user with a role in the given application.
```